ecs-cli/modules/cli/local/converter/converter.go

```diff
@@ -47,26 +46,17 @@ type LinuxParams struct {
 const SecretLabelPrefix = "ecs-local.secret"
 
 // ConvertToDockerCompose creates the payload from an ECS Task Definition to be written as a docker compose file
-func ConvertToDockerCompose(taskDefinition *ecs.TaskDefinition) ([]byte, error) {
+func ConvertToDockerCompose(taskDefinition *ecs.TaskDefinition) ([]composeV3.ServiceConfig, error) {
```

Comment:
What do you think about adding the `localTaskType` and `localTaskValue` as parameters to the function instead? I believe that will get rid of the complexity added in `project.go`:

```go
func ConvertToDockerCompose(taskDefinition *ecs.TaskDefinition, localTaskType, localTaskValue string) ([]byte, error)
```

Reply:
Yeah, that will be better